crates/fluvio/src/producer/accumulator.rs

```diff
@@ -66,9 +67,9 @@ impl RecordAccumulator {
         partition_n: i32,
```

Comment:
I guess that we could change now this to `usize`

Reply:
I tried that but the Encoder and Decoder don't support `usize` and many parts of code get broken. I think that change must be another PR.

Reply:
I am sorry, I mean to change `partition_n: i32` to `partition_n: usize`. I think that it is not used in the fluvio protocol

Reply:
I changed the type, but `PartitionId` and `PartitionCount` are still `i32`.